go/enclave/genesis/testnet_genesis.go

```diff
@@ -0,0 +1,18 @@
+package genesis
+
+import (
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+const TestnetGenesisPK = "8dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b" // The faucet's private key.
```

Comment:
Maybe clearer to call this `TestnetPrefundedPK` or similar?

Reply:
That's actually going away on the next PR. But yes it's clearer.